tools/export/uvision5.py

```diff
@@ -85,17 +87,6 @@ def generate(self, progen_build=False):
         project_data['tool_specific']['uvision5']['misc']['c_flags'].remove("--no_vla")
         project_data['tool_specific']['uvision5']['misc']['ld_flags'] = self.progen_flags['ld_flags']
 
```

Comment:
why was this removed? I cant find anything within the commit message, again multiple changes within one commit and no description. I would guess that this macro is not asm anymore ? :-)